cmd/reports-controller/main.go

```diff
@@ -109,17 +107,6 @@ func createReportControllers(
 				aggregatereportcontroller.Workers,
 			))
 		}
-		if admissionReports {
```

Comment:
Can we also remove the flag?

Reply:
Nope, admission reports still exist (they are now stored in ephemeral reports but the concept is the same).